go.mod

```diff
@@ -5,6 +5,7 @@ go 1.15
 require (
 	github.com/davidlazar/go-crypto v0.0.0-20200604182044-b73af7476f6c // indirect
 	github.com/gogo/protobuf v1.3.2
+	github.com/google/uuid v1.3.0
```

Comment:
nit: Is this change necessary? If not, I'd suggest to revert it to keep the diff with the upstream minimal.